internal/modelcontroller/engine_fasterwhisper.go

```diff
@@ -0,0 +1,156 @@
+package modelcontroller
+
+import (
+	"sort"
+
+	kubeaiv1 "github.com/substratusai/kubeai/api/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/utils/ptr"
+)
+
+func (r *ModelReconciler) fasterWhisperPodForModel(m *kubeaiv1.Model, c ModelConfig) *corev1.Pod {
+	lbs := labelsForModel(m)
+	ann := r.annotationsForModel(m)
+	if _, ok := ann[kubeaiv1.ModelPodPortAnnotation]; !ok {
+		ann[kubeaiv1.ModelPodPortAnnotation] = "8000"
+	}
+
+	args := []string{}
+	args = append(args, m.Spec.Args...)
```

Comment:
nit: why not use `m.Spec.Args` directly in L81?  the slice is not modified

Reply:
Likely due to this being copied from other engine implementations. I will avoid changing in this PR to avoid more scope creep.